src/main/java/org/mockito/internal/configuration/SpyAnnotationEngine.java

```diff
@@ -83,6 +83,7 @@ public AutoCloseable process(Class<?> context, Object testInstance) {
     }
 
     private static Object spyInstance(Field field, Object instance) {
+        // TODO: Should we also add an option to @Spy?
```

Comment:
I did not add an option for now, because it would be the first option on that annotation.